PartyIcons/View/NameplateView.cs

```diff
@@ -224,25 +286,51 @@ public void SetupForPC(XivApi.SafeNamePlateObject npObject)
             }
         }
 
-        private int GetClassIcon(XivApi.SafeNamePlateInfo info)
+        private int GetClassIcon(XivApi.SafeNamePlateInfo info, int def = -1)
         {
+            if (def != -1 && !ignorables.Contains(def))
+                return def;
```

Comment:
Not following the code style, code in the project always have brackets even for a single line statements.